### ISSUE TITLE
nextcloud.export looking for /var/snap/nextcloud/common/nextcloud/data instead of configured directory

### ISSUE THREAD

**Issue by cbeleites** [nextcloud-snap/nextcloud-snap#1961]
#### Describe the bug

`nextcloud.export` stops with error `rsync: change_dir "/var/snap/nextcloud/common/nextcloud/data" failed: No such file or directory (2)`

#### To Reproduce

- I have `datadirectory` configured to `/media/...`
- A backup HDD is mounted to `/var/snap/nextcloud/common/backups/` while doing the backup
- on `nextcloud export` I get:
```
Waiting for MySQL... done
WARNING: This functionality is still experimental and under
development, use at your own risk. Note that the CLI interface is
unstable, so beware if using from within scripts.

Enabling maintenance mode... done
Exporting apps...
        230.50M 100%   23.60MB/s    0:00:09 (xfr#4304, to-chk=0/4735)   
Exporting database...
Exporting config...
Exporting data...
rsync: change_dir "/var/snap/nextcloud/common/nextcloud/data" failed: No such file or directory (2)
              0 100%    0.00kB/s    0:00:00 (xfr#0, to-chk=0/0)
rsync error: some files/attrs were not transferred (see previous errors) (code 23) at main.c(1196) [sender=3.1.2]
Unable to export data
Disabling maintenance mode... done
  ```

#### Expected behavior

`nextcloud.export` should work with other than default data directory configured.

#### Workaround

create a symlink: `/var/snap/nextcloud/common/nextcloud$ ln -s /media/`...`/data`

`nextcloud.export` now works as expected.

#### OS/snapd/snap version
What OS and version are you running? Please also include the output of the
following commands:

    $ snap list nextcloud

```
Name       Version      Rev    Tracking       Publisher   Notes
nextcloud  22.2.3snap1  28662  latest/stable  nextcloud✓  -
```

and:

    $ snap version

```
snap    2.53.4
snapd   2.53.4
series  16
ubuntu  20.04
kernel  5.4.0-1045-raspi
```

#### Logs
Please paste in the output from
[this script](https://github.com/nextcloud/nextcloud-snap/wiki/Issue-Debugging-Scripts)
to assist in debugging (note that it's pre-formatted).

*(will do if needed once the now running backup is finished - not sure whether that'd help since the actual value of `datadirectory` would anyways be `***REMOVED SENSITIVE VALUE***` )*

**Comment by kyrofa** [comment]
Yeah it's true, the data directory is hard-coded to the default in the export job. We need to figure out a way to obtain that information from bash that doesn't mask sensitive values.

**Comment by ppittle** [comment]
I also ran into this issue and am trying the workaround:

>create a symlink: /var/snap/nextcloud/common/nextcloud$ ln -s /media/.../data

Could you add a section to the _Tips & Tricks_ section of the [Wiki](https://github.com/nextcloud-snap/nextcloud-snap/wiki/How-to-backup-your-instance#tips--tricks) on how to use the workaround to perform a backup if you've [moved the data directory](https://github.com/nextcloud-snap/nextcloud-snap/wiki/Change-data-directory-to-use-another-disk-partition)
